tests/kafkatest/services/zookeeper.py

```diff
@@ -51,7 +52,11 @@ def start_node(self, node):
         self.logger.info(config_file)
         node.account.create_file("/mnt/zookeeper.properties", config_file)
 
-        start_cmd = "/opt/%s/bin/zookeeper-server-start.sh " % kafka_dir(node)
+        if ZookeeperService.KAFKA_OPTS:
```

Comment:
Minor - I don't think this conditional is necessary - unless I'm mistaken, exporting an empty `kafka_opts` variable does no harm, so you could always prepend this snippet